charts/istio-authorizer/templates/configmap.yaml

```diff
@@ -5,8 +5,37 @@ metadata:
   labels:
     {{- include "istio-authorizer.labels" . | nindent 4 }}
 data:
+  config.yaml: |
+    logging:
+      level: {{ .Values.logging.level }}
+    discovery:
+      enabled: {{ .Values.discovery.enabled }}
+      namespaces:
+      {{- range .Values.discovery.namespaces }}
+      - {{ . }}
+      {{- end }}
+      timeout: {{ .Values.discovery.timeout }}
```

Comment:
This can probably be refactored to use `with .Values.discovery` to avoid referencing the whole thing in the lines below.

Additional question: don't we just want to embed all props defined under `discovery` instead of listing those one by one?

Reply:
I agree, we could it make more generic.

Reply:
fixed